### PR TITLE
Esc versioning automation api docs

### PR DESCRIPTION
**Pull request by arunkumar611**
Docs that support the new launch blogs

### CODE REVIEW

File: content/docs/esc/environments.md [hunk 1]
```diff
@@ -586,6 +586,70 @@ export const url = functionUrl.functionUrl;
 
 Stacks may only read from environments that belong to the same Pulumi organization.
 
+### With Automation API
+
+You can use ESC with [Automation API](/docs/using-pulumi/automation-api/) in [Node](/docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.Stack.html#addEnvironments), [Go](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3@v3.117.0/go/auto#LocalWorkspace.AddEnvironments), and [Python](docs/reference/pkg/python/pulumi/#pulumi.automation.LocalWorkspace.add_environments). Following methods are supported today:
```
Comment (pgavlin) [inline content/docs/esc/environments.md:591]:
```suggestion
You can use ESC with [Automation API](/docs/using-pulumi/automation-api/) in [Node](/docs/reference/pkg/nodejs/pulumi/pulumi/classes/automation.Stack.html#addEnvironments), [Go](https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3@v3.117.0/go/auto#LocalWorkspace.AddEnvironments), and [Python](docs/reference/pkg/python/pulumi/#pulumi.automation.LocalWorkspace.add_environments). The following methods are supported today:
```

File: content/docs/esc/environments.md [hunk 2]
```diff
@@ -586,6 +586,70 @@ export const url = functionUrl.functionUrl;
...
+
+* `addEnvironments(...)`: Append environments in order to your Pulumi stack's [import](/docs/esc/environments/#using-environments-with-pulumi-iac) list.
```
Comment (pgavlin) [inline content/docs/esc/environments.md:593]:
```suggestion
* `addEnvironments(...)`: Append environments to your Pulumi stack's [import](/docs/esc/environments/#using-environments-with-pulumi-iac) list.
```

File: content/docs/esc/environments.md [hunk 3]
```diff
@@ -586,6 +586,70 @@ export const url = functionUrl.functionUrl;
...
+    +++ myorg/test@2
+...
+```
+
+### Tagging Versions
+
+You can tag your revisions with meaningful names like `Prod`, `Stable`, `v1.1.2`. Each environment has a built-in `latest` tag that always points to the environment’s most recent revision. Use `esc env version tag` to tag a revision. In the following example we are assign `Prod` tag to revision 3 of environment `test`.
+
+```bash
+$ esc env version tag myorg/test@Prod 3
```
Comment (pgavlin) [inline content/docs/esc/environments.md:627]:
```suggestion
$ esc env version tag myorg/test@Prod @3
```

**Review by pgavlin — APPROVED** [review]

**Review by cleverguy25 — APPROVED** [review]

File: content/docs/esc/environments.md [hunk 4]
```diff
@@ -586,6 +586,70 @@ export const url = functionUrl.functionUrl;
...
+ Value
+
+    --- myorg/test@3
+    +++ myorg/test@2
+...
+```
+
+### Tagging Versions
+
+You can tag your revisions with meaningful names like `Prod`, `Stable`, `v1.1.2`. Each environment has a built-in `latest` tag that always points to the environment’s most recent revision. Use `esc env version tag` to tag a revision. In the following example we are assign `Prod` tag to revision 3 of environment `test`.
```
Comment (cleverguy25) [inline content/docs/esc/environments.md:624]: Often tags are lowercase

**Review by cleverguy25 — APPROVED** [review]

**Pull request merged**